shesmu-pluginapi/src/main/java/ca/on/oicr/gsi/shesmu/plugin/filter/BaseRangeActionFilter.java

```diff
@@ -69,4 +69,19 @@ public final void setEnd(Long end) {
   public final void setStart(Long start) {
     this.start = start;
   }
+
+  protected abstract String getName();
```

Comment:
Similarly, would the logging the class name be just as good?

Reply:
Fixed